### DISCUSSION TITLE
Add app lock timer to the Windows desktop app.

### DISCUSSION THREAD

**Discussion by ericsayles** [ente-io/ente#4343] [Enhancements]
I would love the ability to specify an auto-lock timer for the Desktop app. The use case is that I would like the app to enable the App lock after a specified number of minutes or hours after non-use or last unlock. I have other apps that allow me to specify that the app will lock after 4hrs just in case I left it open. The lock on computer lock feature does not appear to work on Windows 10.

**Comment by fleish** [comment]
I would also like this option and/or an option to lock with the computer. I can't speak for the windows app, but on macOS there are no such options currently. "Device lock" doesn't mean the app locks with the device but rather when it is locked it can be unlocked with the user's device credentials vs. a separate password just for the Ente Auth app.
